docs/data/base/components/slider/DiscreteSliderMarks.js

```diff
@@ -109,16 +109,6 @@ const StyledSlider = styled(SliderUnstyled)(
     background-color: #fff;
   }
 
-  & .${sliderUnstyledClasses.valueLabel} {
```

Comment:
These are unnecessary styles applied. The value label is not even shown.

Reply:
Nice catch